src/components/Room/LobbyView/LobbyView.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+import Player from "../../../../server/classes/Player";
+import Room from "../../../../server/classes/Room";
+import IProfile from "../../../../server/interfaces/IProfile";
+import { useSocketRoom } from "../../../hooks";
+import useLocalStorage from "../../../hooks/useLocalStorage";
+import RoomService from "../../../services/RoomService";
+import { Layout, SectionTitle, Title } from "../../Common";
+import Button from "../../Common/Button/Button";
+import UserProfile from "../Id/UserProfile";
+
+export interface LobbyViewProps {
+    room: Room;
+}
+
+export function LobbyView(props: LobbyViewProps): JSX.Element {
+    const socket = useSocketRoom();
+    
+    const [playerId] = useLocalStorage("playerId");
+    
+    const [message, setMessage] = useState("");
+    const [messages, setMessages] = useState([]);
+    const [storageProfile] = useLocalStorage<IProfile>("profile");
+
+    const getRoomId = (): string => {
+        if(typeof window !== "undefined") {
+            return RoomService.getRoomIdFromUrl(window.location.href)
+        }
+        return "";
```

Comment:
Jamais une bonne pratique de retourner un empty string, retourne undefined ?

Reply:
Fixed, can't remember why I used to return empty string. But indeed, bad practice! Thanks!